deploy/charts/rook-ceph/values.yaml

```
@@ -77,9 +77,14 @@ containerSecurityContext:
 # -- If true, loop devices are allowed to be used for osds in test clusters
 allowLoopDevices: false
 
+ceph-csi-operator:
+  nameOverride: ceph-csi-operator
+  fullnameOverride: ceph-csi-operator
```

Can we instead use the shorter prefix? This is more consistent with the manifest install, which just has `ceph-csi` for the service account prefixes. However, in my testing there seems to be a bug with this, i'll comment elsewhere on those details. Maybe you had hit the same issue.
```suggestion
  fullnameOverride: ceph-csi
```

---

acutally we can do but it require extra changes in values.yaml I have added the changes and it works fine